srtp/srtp.c

```diff
@@ -4461,3 +4598,29 @@ srtp_err_status_t srtp_install_log_handler(srtp_log_handler_func_t func, void *
     }
     return srtp_err_status_ok;
 }
+
+srtp_err_status_t
+srtp_set_stream_roc(srtp_t session, uint32_t ssrc, uint32_t roc) {
+    srtp_stream_t stream;
+
+    stream = srtp_get_stream(session, htonl(ssrc));
+    if (stream == NULL)
+        return srtp_err_status_bad_param;
+
```

Comment:
Could potentially check if the current roc is the same as the new roc and so do nothing.... could argue that the application could do the same check.